### ISSUE TITLE
RetroPie-Setup cant restart Samba

### ISSUE THREAD

**Issue by yves84** [RetroPie/RetroPie-Setup#3616]
"Configure Samba ROM Shares" not working.

```
= = = = = = = = = = = = = = = = = = = = =
Installing dependencies for 'samba' : Configure Samba ROM Shares
= = = = = = = = = = = = = = = = = = = = =


= = = = = = = = = = = = = = = = = = = = =
Running action 'install_shares' for 'samba' : Configure Samba ROM Shares
= = = = = = = = = = = = = = = = = = = = =

Failed to restart samba.service: Unit samba.service not found.

```

The samba unit was renamed to `smbd.service`.

```
pi@retropie:~/RetroPie-Setup $ sudo systemctl list-units | grep -i samba
nmbd.service                                                                                                           loaded active running   Samba NMB Daemon                                                                       
smbd.service                                                                                                           loaded active running   Samba SMB Daemon
```

Ty.

**Comment by cmitu** [comment]
> The samba unit was renamed to `smbd.service`.

The configure script tries both `samba` and `smbd` services, if the first fails, it will restart the other. We'll probably remove the old(er) name at some point from the script, but the restart works.

**Comment by cmitu** [comment]
No need to close it, just leave it open until we modify the scriptmodule.

**Comment by cmitu** [comment]
This should be fixed with #3643
